setup.py

```diff
@@ -2,11 +2,13 @@
 
 from distutils.core import setup
 import distutils.core
-import distutils.command.build
-import distutils.command.install_data
+from distutils.command import build, install_data
```

Comment:
IIRC, these are all in `setuptools, too. Switch to it completely rather than using both.